web/src/main/webapp/js/graphhopper/GHRequest.js

```diff
@@ -70,7 +71,7 @@ GHRequest.prototype.init = function (params) {
         else if (val === "true")
             val = true;
 
-        if (key === "point" || key === "mathRandom" || key === "do_zoom" || key === "layer")
+        if (key === "point" || key === "mathRandom" || key === "do_zoom" || key === "layer" || key === "useMiles")
```

Comment:
we should use underscores in all parameters ... not sure why and for what mathRandom was ;)